11-getting-started-with-Jupyter.Rmd

```diff
@@ -0,0 +1,306 @@
+# Getting started with Jupyter {#Getting-started-with-Jupyter}
+
+```{r 11-how-to-get-started-using-r, echo = FALSE, message = FALSE, warning = FALSE}
+library(magick)
+library(magrittr)
+library(knitr)
+knitr::opts_chunk$set(message = FALSE)
+```
+
+## Overview
+
+To be able to effectively and efficiently write R code for your analysis you 
+will need access to the R programming language on a computer. This chapter will 
+show you how to use the R programming language with a common coding 
+platform in data science, Jupyter. In this chapter, we will show you how to use this platform via a web-interface for ease of getting going 
+quickly. In a later chapter, we will also show you how to install and configure this on your local computer (i.e., laptop). These skills are essential to getting your analysis running; think of it like getting dressed in the morning!
+
+## Chapter learning objectives
+
+By the end of the chapter, students will be able to:
+
+- use Jupyter to run, edit and write R code
+- explore plain text data files in Jupyter
+- create new Jupyter notebooks
+- export Jupyter notebooks to other standard file types (e.g., `.html`, `.pdf`) for sharing with non-data science audiences
+
+## Jupyter
+
+Jupyter is a web-based interactive development environment for creating, editing and executing documents called Jupyter notebooks. Jupyter notebooks are documents that contain a mix of computer code (and its output) and formattable text. Given that they combine these two analysis artifacts in a single document—code is not separate from the output or written report—notebooks are one of the leading tools to create reproducible data analyses. Reproducible data analysis is one where you can reliably and easily recreate the same results when analyzing the same data. Although this sounds like something that should always be true of any data analysis, in reality, this is not often the case; one needs to make a conscious effort to perform data analysis in a reproducible manner.
+
+The name Jupyter came from combining the names of the three programming languages that it was initially targeted for (Julia, Python, and R), and now many other languages can be used with Jupyter notebooks.
+
+A Jupyter notebook looks like this:
+
+```{r img-jupyter, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "A screenshot of a Jupyter Notebook.", fig.retina = 2}
+knitr::include_graphics("img/jupyter.png")
+```
+
+### Accessing Jupyter
+
+One of the easiest ways to start computing in R with Jupyter is to use a 
+web-based platform called JupyterHub that already has Jupyter, R, a number of R packages, and collaboration tools installed, configured and ready to use. 
+JupyterHub's are usually created and provisioned by organizations (for example,
+a university, a company, *etcetera*) and require authentication to gain 
+access. <!--- Insert link to public JupyterHub here if we can get 
```

Comment:
I agree, a link to a public JupyterHub where people can follow along with would be very helpful for this chapter!

Reply:
Will add if the collaboration with 2i2c works out!